emannotationschemas/models.py

```diff
@@ -184,22 +162,15 @@ def split_annotation_schema(Schema):
 
     annotation_columns = {}
     segmentation_columns = {}
-    sv_and_root_id_column_keys = get_flat_sv_and_root_id_columns(Schema())
 
     for key, field in flat_schema._declared_fields.items():
         if isinstance(field, mm.fields.Nested):
             raise Exception(f"Schema {flat_schema} must be flattened before splitting")
-        field_type = type(field)
 
-        # pick off the columns which are the supervoxel and root id of
-        # BoundSpatialPoint fields, and separate them as segmentation columns
-        # TODO: make NumericFields illegal and/or fix logic to pick out
-        # actual fields associated with BoundSpatialPoints
-        if key not in sv_and_root_id_column_keys:
-            annotation_columns[key] = field
-        else:
+        if field.metadata.get("segmentation_field"):
```

Comment:
what if the field is missing this metadata?

Reply:
Perhaps should leverage the NumericField, or derive a SegmentationField from that... Otherwise I can make more checks, I'm not happy about this filter either...